src/main/java/mate/academy/internetshop/controllers/CompleteOrderController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.ShoppingCartService;
+import mate.academy.internetshop.service.UserService;
+
+public class CompleteOrderController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+    private final OrderService orderService =
+            (OrderService) INJECTOR.getInstance(OrderService.class);
+    private final UserService userService =
+            (UserService) INJECTOR.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("WEB-INF/views/shoppingcart.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+
+        orderService.completeOrder(shoppingCartService.getByUserId(USER_ID).getProducts(),
+                userService.get(USER_ID));
+        shoppingCartService.clear(shoppingCartService.getByUserId(USER_ID));
+        resp.sendRedirect(req.getContextPath() + "/");
```

Comment:
let's send our user to to this newly created order

Reply:
done